spec/controllers/stalkers_controller_spec.rb

```diff
@@ -1,5 +1,36 @@
 require 'rails_helper'
 
 RSpec.describe StalkersController, type: :controller do
-  pending "Stalkers Controller Tests"
+  describe "GET index " do
+    it "redirects to landing_path if no user id is present" do
+      get :index, stalker_id: 3
+      expect(response).to redirect_to(landing_path)
+    end
+
+    before { request.env["omniauth.auth"] = OmniAuth.config.mock_auth[:twitter]}
+    let!(:stalker) {Stalker.find_or_create_from_auth_hash(OmniAuth.config.mock_auth[:twitter])}
+    it "renders the dashboard_path if user is logged in via twitter" do
+     session[:stalker_id] = 1
+      get :index, stalker_id: 1
```

Comment:
I like to write this as `stalker_id: stalker.id` just in case something changes with the tests down the road.